### PR TITLE
Add a python version of the takehome problem

### PR DESCRIPTION
**Pull request by toofishes**
Should I move these two into separate directories?

I can add README directions as well as long as everyone is OK with this general idea of adding a python version.

### CODE REVIEW

**Comment by toofishes** [comment]
```
$ python3 scoring_test.py
.......
----------------------------------------------------------------------
Ran 7 tests in 0.001s

OK
```

**Comment by bschmeck** [comment]
I would be in favor of splitting the code into `ruby/` and `python/` dirs with a README in each dir.  That will let us adjust the syntax of the examples, etc.  (Right now, everything in the README is in Ruby, which might be confusing for Python devs.)

File: scoring.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Scoring:
+    def score_letter(self, letter):
```
Comment (jcc333) [inline scoring.py]: Doesn't need to be an instance method (I don't know if we're doing code review or if cleaning this up is the candidate's job, etc., but fwiw)

Comment (toofishes) [reply scoring.py]: Just copied ruby from this one- definitely something that can be changed up if desired by the candidate.

**Comment by toofishes** [comment]
This is 100% standard library python 3 code, so no need that I can see for virtualenv.

**Comment by jcc333** [comment]
that's an excellent point; no need to worry about set up if the prerequisites for the set up accomplish the set up.

**Pull request merged**